backend/app.js

```diff
@@ -107,9 +107,13 @@ require('./server/routes/loginRouter')(app)
 require('./server/routes/studentInstanceRouter')(app)
 require('./server/routes/teacherInstanceRouter')(app)
 require('./server/routes/weekRouter')(app)
-app.get('*', (req, res) => res.status(200).send({
-  message: 'Welcome to the beginning of nothingness.',
+app.get('*', (req, res) => res.status(404).send({
+  message: 'Not found.',
 }))
```

Comment:
any path that does not exist or route anywhere should return 404. This 404 response is tested in the routes_spec.js